.github/workflows/presubmit.yml

```diff
@@ -25,10 +25,10 @@ jobs:
       - uses: actions/checkout@v2
       - name: Install clang-tidy
         run: |
-          sudo apt-get install clang-tidy-9
+          sudo apt-get install llvm-9 clang-tidy-9
       - name: Build Compilation DB
         run: |
-          cmake -DCMAKE_BUILD_TYPE=Debug -DCMAKE_EXPORT_COMPILE_COMMANDS=ON -S . -B build
+          cmake -DCMAKE_BUILD_TYPE=Debug -DCMAKE_EXPORT_COMPILE_COMMANDS=ON -DHALIDE_REQUIRE_LLVM_VERSION=90 -S . -B build
```

Comment:
Looks good to me, here. This is probably better than setting LLVM_Dir / Clang_DIR ourselves.